src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php

```diff
@@ -75,12 +75,16 @@ public function buildTemplate()
 
         $this->template = [
             'type' => TemplateType::BUTTONS,
-            'thumbnailImageUrl' => $this->thumbnailImageUrl,
             'title' => $this->title,
             'text' => $this->text,
             'actions' => $actions,
         ];
-
+        
+        /* Make thumbnailImageUrl optional */
+        if(!empty($this->thumbnailImageUrl)){
```

Comment:
Thank you for opening p-r.

The thumbnailImageUrl is string, so please write like
```php
if ($this->thumbnailImageUrl) {
```
I found inconsistent with the official document, the `title` is also optional.
Can I optionalize it incidentally?

I think you need to change constructor as well like this:
```php
public function __construct($title = null, $text, $thumbnailImageUrl = null, array $actionBuilders)
```
This constructor looks ugly though :(

Plus you need to add test cases that you have changed.